.github/workflows/pypi.yaml

```
@@ -0,0 +1,42 @@
+name: Push to Pypi
```

```suggestion
name: Push to PyPI
```

Totally unnecessary, but if you want.